raft/raft.go

```diff
@@ -215,6 +220,12 @@ func (c *Config) validate() error {
 		return errors.New("storage cannot be nil")
 	}
 
+	if c.MaxUncommittedEntriesSize < 0 {
```

Comment:
@nvanbenschoten CI failed due to this checking.

```
raft/raft.go:223:5: unsigned values are never < 0 (SA4003)
```

Reply:
Fixed. CI is passing now.